tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -356,6 +358,31 @@ export class WebGPUBackend extends KernelBackend {
       // Data is on the CPU.
       return info.values;
     }
+
+    const parallelCompilation =
+        env().getBool('WEBGPU_PARALLEL_COMPILATION_PASS') && !this.activeTimers;
```

Comment:
During the lifecycle of a model predict, env().getBool('WEBGPU_PARALLEL_COMPILATION_PASS') should be the same.
So we should make sure this is the same as the one get inside runWebGPUProgram